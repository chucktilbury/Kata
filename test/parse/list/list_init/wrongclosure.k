;-------------------------------------------------------------------------------
; Verify a wrong closure causes error
;-------------------------------------------------------------------------------

var list name = (123, 123, 123)
