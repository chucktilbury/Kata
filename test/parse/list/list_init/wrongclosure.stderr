Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/wrongclosure.k: 5: 22: expected a ')' but got ,
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/wrongclosure.k: 6: 1: expected an assignment item but got end of file
